### ISSUE TITLE
Drop flash and Java applet support in lib/Applet.pm

### ISSUE THREAD

**Issue by taniwallach** [openwebwork/pg#533]
Since modern browsers no longer support Flash or Java applets  - I think we can drop support for such applets in `lib/Applet.pm`.

**Comment by drgrice1** [comment]
This needs to be done, but lets postpone this until after the ww-2.16 release.  This is not urgent.  It is just clean up.

**Comment by taniwallach** [comment]
> This needs to be done, but lets postpone this until after the ww-2.16 release. This is not urgent. It is just clean up.

Agree. I've been testing a relatively strict `Content-Security-Policy` so need to consider what may be loaded from other servers, so am paying more attention to code that loads from external sites... which is why I flagged this. Moved into the "future" project and labeled as "deferred".
